charts/oai-gnb/values.yaml

```diff
@@ -92,7 +92,7 @@ resources:
 ##
 podSecurityContext:
   enabled: false
-  fsGroup: 1001
+  # fsGroup: 1001
```

Comment:
since podSecurityContext.enabled is set to false, commenting this line will have no effect.